kubernetes-sigs/headlamp
# Docker desktop setup instructions and change exposed port

Steps to test
[[Hard Mode]]
- `docker build . -t ghcr.io/kinvolk/headlamp:v0.13.0` in project home directory
- `cd  docker-extension`
- `docker build . -t headlamp-dd`
-  `docker extension install headlamp-dd`
-  Go to docker desktop dashboard and test

[[Easy mode]] 
- `docker extension install yolossn/headlamp-dd-extension`
-  Go to docker desktop dashboard and test

Flows:
1. The user has a `~/.kube/config` file with a proper cluster config -> Headlamp opens to the dashboard UI directly
2. The user has a `~/.kube/config` file but the cluster is not accessible -> The user gets the following message
`Please make sure the Kubernetes cluster is running and accessible, Recieved 'Bad Gateway' when trying to connect`
3. The user doesn't have a `~/.kube/config` file, the user is taken a screen which shows the steps to enable Kubernetes cluster in docker desktop. Once the cluster is up and running and accessible the user is redirected to the dashboard UI.
